connectors/jdbc/element-templates/hybrid/jdbc-outbound-connector-hybrid.json

```diff
@@ -178,27 +194,51 @@
       "type" : "simple"
     },
     "type" : "String"
+  }, {
+    "id" : "connection.databaseName",
+    "label" : "Database name",
+    "optional" : false,
+    "feel" : "optional",
+    "group" : "connection",
+    "binding" : {
+      "name" : "connection.databaseName",
+      "type" : "zeebe:input"
+    },
+    "condition" : {
+      "property" : "connection.authType",
+      "equals" : "detailed",
+      "type" : "simple"
+    },
+    "type" : "String"
+  }, {
+    "id" : "connection.properties",
+    "label" : "Properties",
+    "description" : "Provide the payload for the event as JSON. Details in the <a href=\"https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/amazon-eventbridge/?awseventbridge=outbound\" target=\"_blank\">documentation</a>",
```

Comment:
Description seems to be outdated.

Reply:
That's my bad, I should've mentioned that reviewers shouldn't pay attention to the description for now. I'm writing the documentation, and will add the relevant links where applicable :) 
Same goes for the //TODO